composer.json

```diff
@@ -17,8 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "ext-vips": ">=0.1.2",
+        "php": ">=7.4",
         "psr/log": "^1.1.3|^2.0|^3.0"
     },
     "require-dev": {
```

Comment:
I was able to bump these dev requirements to the latest versions without issues:
```json
        "php-parallel-lint/php-parallel-lint": "^1.3",
        "phpdocumentor/shim": "^3.3",
        "phpunit/phpunit": "^9.5",
        "squizlabs/php_codesniffer": "^3.6"
```

Reply:
nice, done